### ISSUE TITLE
Override description of methods without changing source code

### ISSUE THREAD

**Issue by karolrynio** [springfox/springfox#1541]
- Version: 2.6.0
- Question.

I have spring boot project. Module A is a dependency which is added in `build.gradle` and included to application using `@Import`:

```
@Import({
        ModuleAConfiguration.class
})
```

I have no source code of module A and I cannot modify it, but I want to change description for certain paths, e.g. I want to create `swagger.override.yml`:

```
paths:
  /api/a-module:
    get:
      summary: My custom summary
      description: My custom description
```

and this definition overrides summary and description for path `/api/a-module`.

How to achieve this using springfox?

**Comment by dilipkrish** [comment]
@karolrynio there is a PR #1371  to do just that. Its not been merged in yet.

**Comment by dilipkrish** [comment]
Fixed via #13171
